apps/meteor/tests/end-to-end/api/livechat/09-visitors.ts

```diff
@@ -783,4 +788,125 @@ describe('LIVECHAT - visitors', function () {
 			expect(res.body.visitors[0]).to.have.property('visitorEmails');
 		});
 	});
+	describe('omnichannel/contact', () => {
+		let contact: ILivechatVisitor;
+		it('should fail if user doesnt have view-l-room permission', async () => {
+			await updatePermission('view-l-room', []);
```

Comment:
Cannot use your helpers here? 👀

Reply:
Indeed we should. Done, pls check :heavy_check_mark: